src/display.rs

```diff
@@ -114,21 +114,25 @@ pub fn print_cpu(cpu: &CPU) {
 
 pub fn render_cpu(cpu: &CPU) -> String {
     let temp: colored::ColoredString;
+    // let mut hmm;
```

Comment:
probably remove this, other than that, this looks hella good!

Reply:
oops, that was me testing if it actually worked and i forgot to delete it lol